### ISSUE TITLE
This extension doesn't work when using css.global`...` in external component

### ISSUE THREAD

**Issue by IcyFoxe** [Divlo/vscode-styled-jsx#16]
I've been using external styled-jsx to sepparate my CSS from JSX, by following this guide:
https://github.com/vercel/styled-jsx#external-css-and-styles-outside-of-the-component
However when I'm trying to use this extension with **globally scoped CSS**, this extension doesn't work.

So basically this works:
```
import css from 'styled-jsx/css'

export default css`
  ...
`
```

And this doesn't:
```
import css from 'styled-jsx/css'

export default css.global`
  ...
`
```

Any idea what could be the problem?

**Comment by Divlo** [comment]
Oh indeed, good catch, I just reproduced it, thanks for your report! @IcyFoxe 
I'll try to investigate this and see what's going on tho fix the issue.

**Comment by Divlo** [comment]
@IcyFoxe It should be fixed now in `v1.1.1`, could you confirm it ?
If it is working as expected, you could close this issue.

**Comment by IcyFoxe** [comment]
It works great now. Thank you so much for your work!
